addons/l10n_ec_website_sale/models/payment_method.py

```diff
@@ -0,0 +1,18 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+class PaymentMethod(models.Model):
+    _inherit = 'payment.method'
+
+    l10n_ec_sri_payment_id = fields.Many2one(
+        comodel_name="l10n_ec.sri.payment",
+        string="SRI Payment Method",
+    )
+
+    fiscal_country_codes = fields.Char(compute="_compute_fiscal_country_codes")
```

Comment:
Guessing that you are willing that this field be available to all countries maybe the field should be in a separated module

Reply:
We will probably leave it here until there is a need somewhere else.